### PR TITLE
Refactored the multi-target stuff.

### PR DESCRIPTION
**Pull request by mrstebo**
Just a bit of refactoring and fixing some of the preprocessor directives :+1: 

Also fixed the GitVersion config

### CODE REVIEW

**Comment by ejcoyle88** [comment]
Any idea what the PR branch appveyor error means? 😟 It looks like the branch is broken?

**Comment by mrstebo** [comment]
For some reason this happens when using GitVersion with AppVeyor on PRs :man_shrugging: 

Maybe we should follow this bit of advice: `To disable this error set an environmental variable called IGNORE_NORMALISATION_GIT_HEAD_MOVE to 1`

**Comment by ejcoyle88** [comment]
Yeah, I’m just worried setting that will hide some other problem, but if it
makes the build work it’s worth a go. 🤷‍♀️

I’ll take a look when I get home. 👍


From: Steven Atkinson <notifications@github.com> <notifications@github.com>
Reply: ekmsystems/serilog-enrichers-correlation-id
<reply@reply.github.com>
<reply@reply.github.com>
Date: 30 November 2018 at 20:47:13
To: ekmsystems/serilog-enrichers-correlation-id
<serilog-enrichers-correlation-id@noreply.github.com>
<serilog-enrichers-correlation-id@noreply.github.com>
CC: Elliot Coyle <elliot+github@vaetek.biz> <elliot+github@vaetek.biz>, Review
requested <review_requested@noreply.github.com>
<review_requested@noreply.github.com>
Subject:  Re: [ekmsystems/serilog-enrichers-correlation-id] Refactored the
multi-target stuff. (#8)

For some reason this happens when using GitVersion with AppVeyor on PRs

**Comment by mrstebo** [comment]
Awesome :+1:

File: src/Serilog.Enrichers.CorrelationId.Standard/Extensions/HttpResponseExtensions.cs [hunk 1]
```diff
@@ -1,6 +1,10 @@
-﻿using Microsoft.AspNetCore.Http;
+﻿#if FULLFRAMEWORK
+using System.Web;
+#else
+using Microsoft.AspNetCore.Http;
+#endif
 
-namespace Serilog.Enrichers.CorrelationId.Extensions
+namespace Serilog.Extensions
```
Comment (ejcoyle88) [inline src/Serilog.Enrichers.CorrelationId.Standard/Extensions/HttpResponseExtensions.cs]: Will changing this cause problems for people already using the library?

File: src/Serilog.Enrichers.CorrelationId.Standard/Serilog.Enrichers.CorrelationId.Standard.csproj [hunk 2]
```diff
@@ -4,11 +4,18 @@
     <TargetFramework>netstandard2.0</TargetFramework>
```
Comment (ejcoyle88) [inline src/Serilog.Enrichers.CorrelationId.Standard/Serilog.Enrichers.CorrelationId.Standard.csproj]: Can't we multitarget frameworks using this bit now? Do we need both CSProj?
`<TargetFrameworks>netstandard2.0;v4.5</TargetFramework>`

**Comment by ejcoyle88** [comment]
Also, I took a look at the appveyor errors, but I can't figure out whats up with coveralls. Say's its getting a 422, but then coveralls itself has all the info for the builds as if its worked. Just gonna ignore it for now I guess. :/

**Comment by retro-kharkov** [comment]
Here is a good fix, I have tried to apply nugget pack to my .NET Core 2.1 app, so waiting for seeing this in production

**Comment by mrstebo** [comment]
Moved a bunch of this stuff to #10

**Pull request closed without merging**